ui/pages/extension/AccessMesheryModal.js

```diff
@@ -0,0 +1,106 @@
+import React, {
+} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Grid, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+const styles = makeStyles((theme) => ({
+  paper : {
+    position : 'absolute',
+    width : "30rem",
+    height : "20rem",
+    backgroundColor : theme.palette.background.paper,
+    boxShadow : theme.shadows[5],
+    borderRadius : "4px",
+    top : "0",
+    bottom : "0",
+    left : "0",
+    right : "0",
+    margin : "auto",
+    ["@media (max-width: 455px)"] : {
+      width : "100%"
+    },
+    zIndex : 5
+  },
+  header : {
+    display : "flex",
+    justifyContent : "center",
+    position : "absolute",
```

Comment:
No, sir.

Reply:
It was required here. Also used by MUI dialogs/modals.

Reply:
"used by MUI dialogs/modals." - yes
"It was required here." - no, this is a choice. Other positioning CAN be used.